libs/service-portal/sessions/src/screens/Sessions/Sessions.tsx

```diff
@@ -21,32 +19,50 @@ import LogTable from '../../components/LogTable/LogTable'
 import LogTableMobile from '../../components/LogTable/LogTableMobile'
 import { m } from '../../lib/messages'
 import { useGetSessionsListQuery } from './Sessions.generated'
+import SessionFilter from '../../components/SessionFilter/SessionFilter'
+import addYears from 'date-fns/addYears'
 
 const SESSION_LIMIT = 20
+const initialDates = {
+  fromDate: addYears(new Date(), -1),
+  toDate: new Date(),
+}
 
 const Sessions = () => {
   const { formatMessage } = useLocale()
+  const { lg } = useBreakpoint()
 
   const [sessionsData, setSessionsData] = useState<SessionsSession[]>([])
   const [searchNationalId, setSearchNationalId] = useState('')
   const [nextCursor, setNextCursor] = useState<string>('')
   const [customLoading, setCustomLoading] = React.useState(true)
-
-  const getQueryVariables = (next: string, nationalId: string) => {
+  const [fromDate, setFromDate] = useState<Date>(initialDates.fromDate)
+  const [toDate, setToDate] = useState<Date>(initialDates.toDate)
```

Comment:
~You could have this as an object then instead of 

```
setFromDate(initialDates.fromDate)
setToDate(initialDates.toDate)
```
you would do

```
// Setting the to date
setFilterDates((prev) => ({...prev, to: toDate }))
```